hips/tiles/surveys.py

```diff
@@ -127,7 +132,12 @@ def tile_format(self) -> str:
     @property
     def base_url(self) -> str:
         """HiPS access url"""
-        return self.data['moc_access_url'].rsplit('/', 1)[0]
+        try:
+            return self.data['hips_service_url']
+        except IndexError:
+            return self.data['moc_access_url'].rsplit('/', 1)[0]
+        except:
```

Comment:
I would say: if in doubt, raise an exception, because it's better to fail quickly and loudly than later in weird ways if someone calls this and doesn't handle the error case.